internal/core/src/storage/ChunkCacheSingleton.h

```diff
@@ -39,11 +39,11 @@ class ChunkCacheSingleton {
     }
 
     void
-    Init(const std::string& root_path) {
+    Init(const std::string& root_path, const std::string& madvice) {
```

Comment:
like the constructor, use value type (not reference) and then move them would be more efficient